docs/cloud-native-security/cspm-cloud-posture-dashboard.asciidoc

```diff
@@ -26,14 +26,14 @@ The Cloud Security Posture dashboard shows:
 
 At the top of the dashboard, you can switch between the Cloud accounts and Kubernetes cluster views.
 
-The top section of either view summarizes your overall cloud security posture (CSP) by aggregating data from all monitored resources. Three summary cards on the left show Cloud Accounts or Clusters evaluated, Resources evaluated, and Failed Findings. You can click *Resources Evaluated* or *Failed Findings* to open the <<findings-page, Findings page>>, grouped by resources or filtered for failed findings, respectively.
+The top section of either view summarizes your overall cloud security posture (CSP) by aggregating data from all monitored resources. The summary cards on the left show the number of Cloud Accounts or Clusters evaluated, and the number of Resources evaluated. You can click *Enroll more accounts* or *Enroll more clusters* to deploy to additional cloud assets. 
 
-The remaining summary cards show your overall posture score, and total failed findings grouped by CIS section. Click any CIS section name to view failed findings from that section on the <<findings-page, Findings page>>.
+Click *View all resources* to open the <<findings-page, Findings page>>. The remaining summary cards show your overall posture score, and total failed findings grouped by CIS section. Click *View all failed findings* to view all failed findings, or click a CIS section name to view failed findings from only that section on the Findings page.
```

Comment:
I'd maybe move the first sentence to the previous paragraph, since it relates to a UI link that appears in one of the summary cards described in the previous paragraph.